tests/store/model_registry/test_sqlalchemy_store.py

```diff
@@ -580,3 +582,110 @@ def test_search_registered_model_pagination(self):
             assert exception_context.exception.error_code == ErrorCode.Name(INVALID_PARAMETER_VALUE)
         self.assertIn("Invalid value for request parameter max_results",
                       exception_context.exception.message)
+
+    def test_search_registered_model_order_by(self):
+        rms = []
+        # explicitly mock the creation_timestamps because timestamps seem to be unstable in Windows
+        for i in range(50):
+            with mock.patch("mlflow.store.model_registry.sqlalchemy_store.now", return_value=i):
+                rms.append(self._rm_maker(f"RM{i:03}").name)
+
+        # test flow with fixed max_results and order_by (test stable order across pages)
+        returned_rms = []
+        query = "name LIKE 'RM%'"
```

Comment:
reminder: handle in code and put in tests for `like`, `Like`, `iLike` ... etc

Reply:
Tests for these cases are in #2968 - do you feel it should also be tested in conjunction with the `order_by`?

Also, what do you mean by handle in code?

Reply:
aah! i see. can can keep that PR separate